elements/pfe-tabs/src/pfe-tab.scss

```diff
@@ -273,16 +275,9 @@ $LOCAL-VARIABLES: map-deep-merge($LOCAL-VARIABLES, $backwards-compatibility);
 }
 
 :host([pfe-variant="earth"][on="saturated"][aria-selected="true"]) {
-  color: pfe-local(Color--focus);
-}
-
-:host([pfe-variant="earth"][on="saturated"]:hover) {
-  color: pfe-local(Color--focus);
-}
-
-:host([pfe-variant="earth"][on="saturated"][aria-selected="true"]) {
-  --pfe-tabs--Color: #{pfe-var(text)};
+  --pfe-tabs--BackgroundColor: #{pfe-var(surface--lightest)};
```

Comment:
```suggestion
  --pfe-tabs--BackgroundColor: #{pfe-var(surface--lightest)};
  --pfe-tabs--theme: #{pfe-var(surface--lightest--theme)};
```

Reply:
This will require this be added to `:host`:

```
:host {
   --theme: #{pfe-local(theme)};
}
```

So that theme can pick up the contextual component theme.